test/Data/Graph/Inductive/Query/Properties.hs

```diff
@@ -305,6 +341,15 @@ test_msTree _ cg = ns == mstNs && S.isSubsetOf mstEs es
     toE (w,l) (v,_) = (v,w,l)
 
 -- -----------------------------------------------------------------------------
+-- SCC
+
+-- | The strongly connected components should be a partitioning of the nodes of
+--   a graph.
+test_strongComponentsOf :: (Graph gr) => Proxy (gr a b) -> gr a b -> Bool
+test_strongComponentsOf _ g =
+  sort (concatMap nodes (strongComponentsOf g)) == sort (nodes g)
+
```

Comment:
Are you able to add a property as to the behaviour of the results of strongComponentsOf (e.g. disjoint)?

Reply:
Test case has been expanded with additional property checks.